model_bakery/baker.py

```diff
@@ -916,4 +947,7 @@ def bulk_create(baker: Baker[M], quantity: int, **kwargs) -> list[M]:
                         kwargs[reverse_relation_name]
                     )
 
+        if _full_clean:
+            entry.full_clean()
```

Comment:
I think, this one is being called too late. We already created objects in line 918: `created_entries = manager.bulk_create(entries)`, and only now doing a validation.  This does not prevent objects from being saved to the db.

As an alternative, we could pass it in baker.prepare a bit higher in the stack:

```
entries = [
    baker.prepare(
        _full_clean=_full_clean,
        **kwargs,
    )
    for _ in range(quantity)
]
```